app/(auth)/signin/page.tsx

```diff
@@ -57,32 +66,109 @@ export default function SignIn() {
   })
 
   return (
-    <>
-      <Heading>Welcome Back!</Heading>
-      <Heading>Sign In</Heading>
-      <Box as="form" onSubmit={signInHandler}>
-        <Flex flexDirection={'column'}>
-          <FormControl isInvalid={!!errors.email}>
-            <FormLabel>Email address</FormLabel>
-            <Input {...register('email')} />
-            {errors.email && (
-              <FormErrorMessage>{errors.email.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <FormControl isInvalid={!!errors.password}>
-            <FormLabel>Password</FormLabel>
-            <Input type="password" {...register('password')} />
-            {errors.password && (
-              <FormErrorMessage>{errors.password.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <PrimaryButton isLoading={isLoading} type={'submit'}>
-            Sign In
-          </PrimaryButton>
-          <Heading>{"Don't"} have an account yet?</Heading>
-          <Link href="/signup/email">Sign Up</Link>
-        </Flex>
-      </Box>
-    </>
+    <Box
+      as="main"
+      h="100vh"
+      w="100vw"
+      bg={{
+        base: 'white',
+        md: 'linear-gradient(242.77deg, #C4E6E6 1.15%, #085783 100%)'
+      }}
+      color="black"
+    >
+      <AbsoluteCenter>
```

Comment:
For PC, it is center, but for SP it is not the center. For example, the iPhone 12 Pro is too below (because it is center). Could you adjust with padding for SP? Padding-top should be 40px.
<img width="323" alt="Screenshot 2024-02-04 at 11 16 30 AM" src="https://github.com/tabi-memo/frontend/assets/66394413/718d2125-d50f-4afb-9549-b8c2f32eb530">

Reply:
Added top 40px on mobile size!

<img src="https://github.com/tabi-memo/frontend/assets/70562492/9b98f910-c001-42cb-900d-0fdb7dac354e" width="300" />